### PR TITLE
cleanup `AepUi`, add unit tests

### PR DESCRIPTION
**Pull request by rymorale**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/AepUi.kt [hunk 1]
```diff
@@ -16,14 +16,14 @@ import com.adobe.marketing.mobile.aepuitemplates.AepUITemplate
...
- * This interface can be implemented by various UI components like [SmallImageCardAepUi], [LargeImageCardAepUi], etc.
+ * This interface can be implemented by various UI components like [SmallImageUi], LargeImageUi, etc.
  * It associates a specific template and state to ensure consistency when rendering and managing UI components.
  *
  * @param T The type of the template associated with the UI component.
  * @param S The type of the state associated with the UI component.
  */
 
-sealed interface AepUI<T : AepUITemplate, S : AepCardUIState> {
+sealed interface AepUi<T : AepUITemplate, S : AepCardUIState> {
```
Comment (prudrabhat) [inline code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/AepUi.kt:26]: This should be left as it was based on [Kotlin naming conventions for acronyms](https://kotlinlang.org/docs/coding-conventions.html#choose-good-names)

_When using an acronym as part of a declaration name, capitalize it if it consists of two letters (IOStream); capitalize only the first letter if it is longer (XmlFormatter, HttpInputStream)._

Comment (rymorale) [reply code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/AepUi.kt:26]: ah my bad, will revert the name change

**Review by prudrabhat — APPROVED** [review]

File: code/messaging/src/test/java/com/adobe/marketing/mobile/aepcomposeui/aepui/SmallImageUiTests.kt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import com.adobe.marketing.mobile.aepcomposeui.aepui.state.SmallImageCardUIState
+import com.adobe.marketing.mobile.aepuitemplates.SmallImageTemplate
+import com.adobe.marketing.mobile.aepuitemplates.uimodels.AepText
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class SmallImageUiTests {
+
+    @Test
+    fun test_SmallImageUi_initialState() {
```
Comment (prudrabhat) [inline code/messaging/src/test/java/com/adobe/marketing/mobile/aepcomposeui/aepui/SmallImageUiTests.kt:23]: Can you rename the tests to  be descriptive as Kotlin allows it  - 
```
fun ` Test Initial State ... `()
```

**Review by spoorthipujariadobe — APPROVED** [review]

File: code/messaging/src/test/java/com/adobe/marketing/mobile/aepcomposeui/aepui/SmallImageUiTests.kt (hunk 2 above)
Comment (rymorale) [reply code/messaging/src/test/java/com/adobe/marketing/mobile/aepcomposeui/aepui/SmallImageUiTests.kt:23]: i'll update these test names and the previously added unit test names in a coming pr

Comment (spoorthipujariadobe) [reply code/messaging/src/test/java/com/adobe/marketing/mobile/aepcomposeui/aepui/SmallImageUiTests.kt:23]: Got it in my PR when I was resolving conflicts :)

**Pull request merged**